.travis.yml

```diff
@@ -54,14 +54,13 @@ script:
   #   * visualization_test
   #   * model_rl_experiment_test
   # algorithmic_math_test: flaky
-  # r_transformer_test: requires new feature in tf.foldl (rm with TF 1.9)
-  - pytest
```

Comment:
Hi @MostafaDehghani -- Can you put this "- pytest" line back ... I've worked out how to import this automatically internally.

Thanks again for your patience as I continue to work on the tooling!

Reply:
Done! :)